storage/tianmu/system/tianmu_file.cpp

```diff
@@ -52,23 +54,23 @@ int TianmuFile::Flush() {
 }
 
 int TianmuFile::OpenCreate(std::string const &file) {
-  return Open(file, O_CREAT | O_RDWR | O_LARGEFILE | O_BINARY, tianmu_umask);
+  return Open(file, O_CREAT | O_RDWR | O_LARGEFILE | O_BINARY, TIANMU_UMASK);
```

Comment:
Is `TIANMU_UMASK` a constant name? or an enum? Fix it with google style:
https://google.github.io/styleguide/cppguide.html#Constant_Names

Reply:
is Macro